.eslintrc

```diff
@@ -60,7 +60,6 @@ rules:
   indent: [2, 2, {SwitchCase: 1}]
   key-spacing: [2, {mode: "minimum"}]
   keyword-spacing: 2
-  linebreak-style: [2, "unix"]
```

Comment:
Your choices with this rule are `unix`, `windows`, or disabling/removing the rule. If you want to leave the line there and have the rule explicitly disabled, change `2` to `0`. We don't do that for any other rules, though, if that matters.